stream.c

```diff
@@ -427,6 +427,7 @@ static void* handle_md5_digest(void* param)
 #define SERVER_URI_LEN 512
     char server_uri[SERVER_URI_LEN];
     char* server_user = NULL, *server_pass = NULL;
+    unsigned int server_pass_sz;
```

Comment:
This should be a `size_t` type in order to match the later `strlen` call. In addition, add a `#include <string.h>` line which appears to be missing in this file.